creme/mobile/templatetags/mobile_tags.py

```diff
@@ -31,13 +31,32 @@
     REL_SUB_ACTIVITY_SUBJECT,
     STATUS_IN_PROGRESS,
 )
-# from creme.persons import constants as persons_constants
+from creme.creme_core.utils.settings import get_setting_value
+from creme.mobile import setting_keys
 from creme.persons import get_organisation_model
 
+# from creme.persons import constants as persons_constants
```

Comment:
It was less ugly at its previous place (I'll clean all this commented code at the new release begining).